### PR TITLE
Docs(web-react): Introduce Migration Guide to version 3

### PR DESCRIPTION
**Pull request by crishpeen**
### Issue reference

https://jira.almacareer.tech/browse/DS-1492

### CODE REVIEW

File: docs/migrations/web-react/MIGRATION-v3.md [hunk 1]
```diff
@@ -38,7 +46,17 @@ Manually replace the props in your project.
 - `<ButtonLink isSquare … />` → `<ButtonLink isSymmetrical … />`
 </details>
 
-### Heading elementType prop is now mandatory
+### Button and ButtonLink: Removed `inverted` Variant
+
+The `inverted` variant was removed from the `Button` and `ButtonLink` components.
+Use themes instead to switch the color scheme.
```
Comment (literat) [inline docs/migrations/web-react/MIGRATION-v3.md:52]: Can we link to an additional migration guide?

File: packages/codemods/src/transforms/v3/web-react/__testfixtures__/toastbar-inverted-neutral.input.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// @ts-ignore: No declaration -- The library is not installed; we don't need to install it for fixtures.
```
Comment (literat) [inline packages/codemods/src/transforms/v3/web-react/__testfixtures__/toastbar-inverted-neutral.input.tsx:1]: Do we not ignore these fixtures by the TypeScript compiler?

File: docs/migrations/web-react/MIGRATION-v3.md [hunk 3]
```diff
@@ -86,6 +104,50 @@ Manually replace the props in your project.
...
+### Pill: Removed Action Colors from `color` Prop
+
+The `color` prop of the `Pill` component no longer supports `primary`, `secondary`, `tertiary`, `inverted`
+and `unselected` variants. Instead, the `neutral` variant was added. The current list of variants is:
+
+- `neutral`
+- `danger`
+- `informative`
+- `success`
+- `warning`
```
Comment (adamkudrna) [inline docs/migrations/web-react/MIGRATION-v3.md:121]: I'd order these either alphabetically or logically (neutral, <emotion>).

File: docs/migrations/web-react/MIGRATION-v3.md [hunk 4]
```diff
@@ -86,6 +104,50 @@ Manually replace the props in your project.
...
+
+### ProductLogo: Removed `inverted` Variant
+
+The `inverted` variant was removed from the `ProductLogo` component.
+Use themes instead to switch the color scheme.
```
Comment (adamkudrna) [inline docs/migrations/web-react/MIGRATION-v3.md:126]: I think themes will not help here, `ProductLogo` is now independent on themes. Users may want to switch the actual image source though.

File: docs/migrations/web-react/MIGRATION-v3.md [hunk 5]
```diff
@@ -86,6 +104,50 @@ Manually replace the props in your project.
 - `<Link isUnderlined … />` → `<Link underlined="always" … />`
 </details>
 
+### Link: Removed `inverted` Variant
+
+The `inverted` variant was removed from the `Link` component.
+Use themes instead to switch the color scheme.
+
+### Pill: Removed Action Colors from `color` Prop
```
Comment (adamkudrna) [inline docs/migrations/web-react/MIGRATION-v3.md:112]:
```suggestion
### Pill: Update Color Palette
```

Or similar? Becaouse not only action colors have been removed.

**Review by adamkudrna — APPROVED** [review]
I didn't check if the list of changes is complete, but what's here looks good.

File: docs/migrations/web-react/MIGRATION-v3.md (hunk 3 above)
Comment (crishpeen) [reply docs/migrations/web-react/MIGRATION-v3.md:121]: Neutral is first and then the rest is alphabetically.

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**